web/src/stream_color.js

```diff
@@ -39,7 +39,11 @@ export function get_corrected_color(color) {
 }
 
 export function get_stream_privacy_icon_color(color) {
-    return get_corrected_color(color).toHex();
+    const corrected_color = get_corrected_color(color);
+    if (settings_data.using_dark_theme()) {
+        corrected_color.saturate(0.3).lighten(0.1).toHex();
+    }
+    return corrected_color.darken(0.12).toHex();
```

Comment:
In dark mode this will do `.darken(0.12)`, after doing `.saturate(0.3)` and `.lighten(0.1)`. I don't think that's intentional.

Reply:
Thanks for pointing that out! Fixed.